[IMP] payment_providers: add SEPA provider and adapt related docs (#12825)
odoo/documentation: content/applications/finance/payment_providers/wire_transfer.rst

Wire transfers
==============

The **Wire Transfer** payment method allows you to provide payment instructions to your customers,
such as your bank details and a reference to include with the payment. These instructions are
displayed once the customer selects :guilabel:`Wire Transfer` as the payment method and clicks the
:guilabel:`Pay now` button at the end of the checkout process on your ecommerce website or
on the customer portal.

> .. image:: wire_transfer/payment_instructions_portal.png

Reviewer: This image looks outdated to me. Here's what I'm finding in 17.0:
![image](https://github.com/user-attachments/assets/c9ce47c0-66d8-44ab-b8a1-d6746d14375e)

Wire transfers
==============

The **Wire Transfer** payment method allows you to provide payment instructions to your customers,
such as your bank details and a reference to include with the payment. These instructions are

> displayed once the customer selects :guilabel:`Wire Transfer` as the payment method and clicks the

Reviewer 2: ```suggestion
displayed after the customer selects :guilabel:`Wire Transfer` as the payment method and clicks the
```

:guilabel:`Pay now` button at the end of the checkout process on your ecommerce website or
on the customer portal.

.. image:: wire_transfer/payment_instructions_portal.png
   :scale: 80%
   :alt: Payment instructions on the customer portal

.. note::
   - While this method is widely accessible and requires minimal setup, it is not efficient
     process-wise. It is recommended to set up a :doc:`payment provider <../payment_providers>`
     instead.
   - Online orders remain in the :guilabel:`Quotation sent` (i.e., unpaid order) stage until you

>      receive the payment and :guilabel:`Confirm` the order.

Reviewer 2: and *manually* confirm?

Configuration
=============

To configure **Wire Transfer**:

#. :ref:`Navigate to the Wire Transfer payment provider <payment_providers/supported_providers>`.

> #. In the :guilabel:`Configuration` tab, select the :guilabel:`Communication` to be used:

Reviewer: Since "Communication" isn't used like this in English (more often "Memo"), it wasn't immediately clear to me what this would do. I think that it would be more clear like this (but feel free to edit as you see fit):
```suggestion
#. In the :guilabel:`Configuration` tab, select whether the memo or :guilabel:`Communication` to be displayed alongside the payment instructions should be:
```

Reviewer: IMPORTANT - this section is duplicated in the SEPA Direct Debits file, so make sure to update it there also :)

>    - :guilabel:`Based on Document Reference`: sales order or invoice number
>    - :guilabel:`Based on Customer ID`: customer identifier

Reviewer: ```suggestion
   - :guilabel:`Based on Document Reference`: the sales order or invoice number
   - :guilabel:`Based on Customer ID`: the customer identifier
```
This makes it work better with the sentence I suggested above. If you decide to go with something else, this may not be necessary.

Author: haha so funny, I actually did this while applying your previous suggestion, before I saw this comment :smile:

Reviewer: What a couple of geniuses we are :sunglasses:
